tests/unit/actor/test_actor_create_request_list.py

```diff
@@ -0,0 +1,204 @@
+from __future__ import annotations
+
+import re
+import typing
+from unittest import mock
+from unittest.mock import call
+
+import pytest
+
+from crawlee._request import UserData  #  TODO: Make public in Crawlee?
+from crawlee._types import HttpHeaders, HttpMethod  #  TODO: Make public in Crawlee?
```

Comment:
What do you mean, make them public? If you do for example this:

```py
class A(BaseModel):
  method: HttpMethod
```

and `A` gets exported... that's fine. If you just need to export it so that the end user can do `from apify import HttpMethod`, then I'm not sure if it's a good idea.

Reply:
I mean that in apify-sdk-python I use internal crawlee types. So either I expose those types in crawlee or I will keep using internal types of another module, which is also not very good.

So I would like the user to be able to do 
from crawlee import HttpMethod
instead of 
from crawlee._types import HttpMethod

Reply:
Keep reaching inside crawlee. It's better to make an exception for ourselves instead of committing to maintaining BC with stuff that we only need to make for SDK by making it public.